### PR TITLE
Update examples

### PR DESCRIPTION
**Pull request by giovannicuriel**
This PR updates the examples given in the documentation.
  
This is part of dojot/dojot#143

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 [![License badge](https://img.shields.io/badge/license-GPL-blue.svg)](https://opensource.org/licenses/GPL-3.0)
 [![Docker badge](https://img.shields.io/docker/pulls/dojot/iotagent-json.svg)](https://hub.docker.com/r/dojot/device-manager/)
+[![Travis badge](https://travis-ci.org/dojot/device-manager.svg?branch=master)](https://travis-ci.org/dojot/device-manager#)
```
Comment (mmagr) [inline README.md]: mmm, we're actually just building docs, so this can be misleading - but ok

File: README.md [hunk 2]
```diff
@@ -24,7 +24,7 @@ But you won't need to worry about installing any of these - they are automatical
 
 ## How to run
 
-If you really need to run DeviceManager as a standalone process (without dojot's wonderful docker-compose), you can execute these commands:
+If you really need to run DeviceManager as a standalone process (without dojot's wonderful [docker-compose](https://github.com/dojot/docker-compose)), you can execute these commands:
```
Comment (mmagr) [inline README.md]: I think that a note of warning related to user management is due here.
That should be specially important if we transition to a more strict checking of tokens on services (i.e. make proper use of the "dummy" strings at the beginning and end of the token)

File: README.md [hunk 3]
```diff
@@ -23,7 +24,7 @@ But you won't need to worry about installing any of these - they are automatical
 
 ## How to run
```
Comment (mmagr) [inline README.md]: mmm, if we're really keeping this, we should remind the user of the need to have a running instance of postgres, should we not?

Comment (mmagr) [reply README.md]: Also, a note of warning due to apigw mediation might come in handy - when using the service from a dojot instance, the user should never be exposed to internal service addresses and ports. All examples assume the "standalone" execution of the service, and the user might feel lost to the changes in baseurl address (tcp port, more precisely) between apib and this.

Comment (giovannicuriel) [reply README.md]: Ok! I'll add a note related to postgres and add a section (or something alike) indicating how the user should send the requests using a full dojot setup.

File: README.md (hunk 2 above)
Comment (giovannicuriel) [reply README.md]: I'll add a comment on that.

File: README.md (hunk 1 above)
Comment (giovannicuriel) [reply README.md]: Travis has built the docs correctly :)
I agree - this badge should be here only if there's something that could go wrong and affect others.

**Pull request merged**